demo/java-demo-client/src/main/resources/templates/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+	<head>
+		<title>CP Metering on Cloud Foundry - Simple Demo</title>
+		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+		<script id="sap-ui-bootstrap" type="text/javascript"
```

Comment:
I can see that OpenUI5 is Apache2 so for the first version this would do. However we should unify the dashboard and this UI to use the same technology. We should file a github issue to cover the work we need to do here.